requirements.txt

```diff
@@ -1,13 +1,13 @@
 GitPython==2.0.6
 OWSLib
-geopy==1.11.0
```

Comment:
you don't need to modify this file, it has it's own requirements.txt file https://github.com/aodn/data-services/blob/master/AODN/AODN-WAVE-DM/requirements.txt